src/rp/util/string.clj

```diff
@@ -28,3 +28,8 @@
   (some-> s
           parse-double*
           double))
+
+(defn non-blank-string?
+  "Returns true if x is a non-blank string"
+  [x] (and (string? x)
```

Comment:
:+1: I've seen this style in Clojure's own code base when the body is a single, trivial expression, but in general I find the extra break helpful for clearly discerning args from body (especially if the args are complex or include destructuring).